backstop/scenarios/header.js

```diff
@@ -21,4 +21,14 @@ module.exports = [
     actions: [hover('#test-button-3'), focus('#test-button-4')],
     selectors: ['#demo-slim'],
   }),
+
+  //// Split
+  scenario('Type Default State hover & focus', {
```

Comment:
Wait, why our different scenario names are same???

Reply:
Changed scenario names

Reply:
Looks like the images changed file names because of that. Can you delete old images?

One way to do that is to run `yarn test:ci` (without any filter) and then `yarn clean:images`.